R/recipe_run.R

```diff
@@ -381,6 +381,25 @@ recipe_prepare_workdir <- function(info) {
     orderly_log("depends", str)
     file_copy(src, dst)
   }
+  
+  if (!is.null(info$global_resources)) {
```

Comment:
This needs moving about:

* the check that the `global_resources` entry exists and is a directory belongs in the config (see `orderly_config_read_yaml`)
* the there needs to be a check here that all the global resources in _this_ orderly report actually exist (that check is missing at the moment I believe)